src/pages/MarketPage/components/SalesProducts/SalesProducts.tsx

```diff
@@ -24,12 +33,17 @@ function SalesProducts() {
   const [orderBy, setOrderBy] = useState("recent");
   const [pageSize, setPageSize] = useState(getHtmlSize());
   const [page, setPage] = useState(1);
-  const [totalPageNum, setTotalPageNum] = useState();
+  const [totalPageNum, setTotalPageNum] = useState<number>();
```

Comment:
```suggestion
  const [totalPageNum, setTotalPageNum] = useState<number>(여기에);
```
useState(여기에) number type의 초깃값을 넣어주세요!